ts/Extensions/Annotations/Popup.ts

```diff
@@ -686,10 +686,14 @@ H.Popup.prototype = {
             });
 
             if (isRoot) {
-                storage = storage.sort(function (a): number {
-                    return (a as any)[1].match(/format/g) ? -1 : 1;
+                H.stableSort(storage, function (a: any): number {
```

Comment:
Instead of referencing `H.stableSort`, get it from the Utilities directly. See for example Legend.ts:
```js
import U from './Utilities.js';
const {
    [...],
    stableSort,
    [...]
} = U;
```